man/systemd-mount.xml

```diff
@@ -289,6 +289,9 @@
         <listitem>
           <para>Create and mount a new temporary file system on <replaceable>WHERE</replaceable>, with an
           optional <replaceable>NAME</replaceable> that defaults to <literal>tmpfs</literal>.</para>
+
+          <para>The file system is mounted with the top-level directory mode determined by the umask setting
+          of the caller. This is different from the kernel default of <literal>rwxrwxrwxt</literal>.</para>
```

Comment:
hmm, i'd word this differently, i.e. say "`0777`, masked by the umask of the caller`. And maybe make umask a link to umask(2) or so. And maybe even compare this with mkdir(1) which does the same.

Also, maybe mention the sticky bit explicitly. The `t` is not easy to see and not everyone will grok this. Hence maybe push people explicit to it. maybe just suffix with `(i.e. kernel sets sticky bit!)`